ledger-core/virtual/integration/outgoing_test.go

```diff
@@ -201,10 +201,11 @@ func TestVirtual_CallMethodOutgoing_WithTwicePulseChange(t *testing.T) {
 				assert.Equal(t, firstExpectedToken, request.DelegationSpec)
 				assert.NotEqual(t, payload.RepeatedCall, request.CallRequestFlags.GetRepeatedCall())
 
-				server.IncrementPulseAndWaitIdle(ctx)
+				server.IncrementPulse(ctx)
```

Comment:
maybe change for `secondPulse`, too (line 136)

Reply:
and in TestVirtual_CallMethodOutgoing_WithTwicePulseChange, too

Reply:
fixed